Earthfile

```diff
@@ -230,6 +230,7 @@ unit-test:
     END
     BUILD ./ast+unit-test
     BUILD ./util/deltautil+unit-test
+    RUN go test ./conslogging
```

Comment:
This should be handled by the `ARG pkgname = ./...` (used by `./not-a-unit-test.sh`). The only reason that `ast` and `util/deltautil` are run separately is that they are separate go modules (i.e. they have their own `go.mod` and `go.sum`), so `./...` skips them (among a few other problems).

Reply:
Oh - I didn't see `pkgname` used and the whole `not-a-unit-test.sh` didn't make it obvious. 

Could we pass it explicitly to make it more obvious?

Reply:
Yeah, I think that would be a good change.

Reply:
It might also be worth commenting above `BUILD ./ast+unit-test` something like
```
# The following are separate go modules and need to be tested separately.
```

Reply:
done and added comment